### ISSUE TITLE
[BUG] - Date Range Picker TypeError (Error on click the right corner)

### ISSUE THREAD

**Issue by DanielReyes03** [nextui-org/nextui#3008]
### NextUI Version

2.3.6

### Describe the bug

When i clicked the bottom right corner of the date range picker before or after select a new range of dates. The component fallbacks an error

``` js
Unhandled Runtime Error
TypeError: Failed to execute 'createTreeWalker' on 'Document': parameter 1 is not of type 'Node'.
```

This occurs on the main website and when using the component in my own project

### Your Example Website or App

https://nextui.org/docs/components/date-range-picker

### Steps to Reproduce the Bug or Issue

1. Go to the Date range picker
2. Click in the bottom right corner of the component

### Expected behavior

As a user, i don't expect to have this error on the component.

### Screenshots or Videos

https://github.com/nextui-org/nextui/assets/58609880/bfcbe55a-d70f-4113-8237-0268321d6873

### Operating System Version

macOS

### Browser

Chrome

**Comment by azharisubroto** [comment]
I have the same issue

**Comment by zillBoy** [comment]
Hi, I also received the same issue in my next.js project using `"@nextui-org/react": "^2.3.6"`

```
Unhandled Runtime Error
TypeError: Failed to execute 'createTreeWalker' on 'Document': parameter 1 is not of type 'Node'.
```

The way I solved it was by installing the package as individual `npm install @nextui-org/date-picker`, and then using that import in my project.

 Before I was using global `import {DateRangePicker} from "@nextui-org/react";` which gave me the error, thank you

**Comment by wingkwong** [comment]
@zillBoy Just FYI. This issue has been fixed and our latest version is v2.4.2.
